From 8cceda8f469b87de708545525824d2d6c025dbfa Mon Sep 17 00:00:00 2001
From: Patrick Lower <devvortex@gmail.com>
Date: Fri, 27 Feb 2015 00:50:46 -0500
Subject: [PATCH 03/25] input: synaptics: Disable stylus mode

* Fixes touchscreen bug observed in some games, emulators, and apps.

Change-Id: I7628366d6f7fa8824b1c255031c34d06a9570565
[haggertk: Clarification, this is for klte devices only - K_PROJECT,
 KACTIVE_PROJECT, KSPORTS_PROJECT]
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 drivers/input/touchscreen/synaptics/synaptics_i2c_rmi.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/input/touchscreen/synaptics/synaptics_i2c_rmi.h b/drivers/input/touchscreen/synaptics/synaptics_i2c_rmi.h
index 2b46f4568e6..ea0a99fc70b 100644
--- a/drivers/input/touchscreen/synaptics/synaptics_i2c_rmi.h
+++ b/drivers/input/touchscreen/synaptics/synaptics_i2c_rmi.h
@@ -90,7 +90,6 @@
 #define USE_ACTIVE_REPORT_RATE
 #define USE_F51_OFFSET_CALCULATE
 #define SYNAPTICS_DEVICE_NAME	"G900F"
-#define USE_STYLUS
 #define USE_DETECTION_FLAG2
 #define USE_EDGE_EXCLUSION
 
@@ -105,7 +104,6 @@
 #define USE_ACTIVE_REPORT_RATE
 #define USE_F51_OFFSET_CALCULATE
 #define SYNAPTICS_DEVICE_NAME	"G870"
-#define USE_STYLUS
 #define USE_DETECTION_FLAG2
 #define USE_EDGE_EXCLUSION
 
@@ -120,7 +118,6 @@
 #define USE_ACTIVE_REPORT_RATE
 #define USE_F51_OFFSET_CALCULATE
 #define SYNAPTICS_DEVICE_NAME	"G860"
-#define USE_STYLUS
 #define USE_DETECTION_FLAG2
 #define USE_EDGE_EXCLUSION
 
-- 
2.17.1

